static/adopters.csv

```diff
@@ -90,6 +90,7 @@ First Contributions, https://github.com/firstcontributions/first-contributions
 Freedomotic, https://github.com/freedomotic/freedomotic
 friends, https://github.com/JacobEvelyn/friends
 Fukuzatsu, https://gitlab.com/coraline/fukuzatsu/tree/master
+funKounter, https://github.com/lucliscio/fankounter
```

Comment:
Is the name `funKounter` spelled correctly?
I suppose it should be called `fanKounter`.

Reply:
yes the correct name is fanKounter.... I apologize for my mistake

Reply:
Esta mal